sklearn/model_selection/_split.py

```diff
@@ -543,11 +543,11 @@ class StratifiedKFold(_BaseKFold):
     >>> from sklearn.model_selection import StratifiedKFold
     >>> X = np.array([[1, 2], [3, 4], [1, 2], [3, 4]])
     >>> y = np.array([0, 0, 1, 1])
-    >>> skf = StratifiedKFold(n_splits=2)
+    >>> skf = StratifiedKFold(n_splits=2, random_state=0)
```

Comment:
Which is why I wanted to preserve the user-set random state by preserving it in the original `random_state` variable. You suggested that it would be clear to the user to display the explicitly set `random_state` and hence the current setup...